docs/pages/api/inputs/mp4.md

```diff
@@ -0,0 +1,6 @@
+import Docs from "@site/pages/api/generated/renderer-Mp4.md"
+
+# MP4
+MP4 inputs allow the compositor to use video and audio from MP4 files in scenes.
```

Comment:
```suggestion
An Input type that allows the compositor to read static MP4 files.
```

Reply:
Also add information what is supported, what choices we are making when there is an option. e.g.
- what codecs we are supporting (audio + video)
- how we are selecting tracks when there is more than one
...